circom-types/src/plonk/zkey.rs

```diff
@@ -261,20 +257,20 @@ where
         let mut lagrange = None;
         let mut p_tau = None;
         rayon::scope(|s| {
-            s.spawn(|_| additions = Some(Self::additions_indices(n_additions, &mut add_section)));
-            s.spawn(|_| map_a = Some(Self::id_map(n_constraints, &mut a_section)));
-            s.spawn(|_| map_b = Some(Self::id_map(n_constraints, &mut b_section)));
-            s.spawn(|_| map_c = Some(Self::id_map(n_constraints, &mut c_section)));
-            s.spawn(|_| qm = Some(Self::evaluations(domain_size, &mut qm_section)));
-            s.spawn(|_| ql = Some(Self::evaluations(domain_size, &mut ql_section)));
-            s.spawn(|_| qr = Some(Self::evaluations(domain_size, &mut qr_section)));
-            s.spawn(|_| q0 = Some(Self::evaluations(domain_size, &mut q0_section)));
-            s.spawn(|_| qc = Some(Self::evaluations(domain_size, &mut qc_section)));
+            s.spawn(|_| additions = Some(Self::additions_indices(n_additions, add_section)));
```

Comment:
Das könnte maybe auch ein eigener commit sein in der Zukunft
refactor: removed unnecessary muts in zkey parsing

ist aber egal